app/models/schedule.rb

```diff
@@ -35,7 +35,11 @@ class Schedule < ApplicationRecord
               scope: :contract_period_year
             }
 
-  def teacher_type
-    @teacher_type ||= MENTOR_TEACHER_TYPE_IDENTIFIERS.include?(identifier) ? :mentor : :ect
+  scope :excluding_replacement_schedules, -> {
+    where.not("identifier IN (?)", REPLACEMENT_SCHEDULE_IDENTIFIERS)
```

Comment:
Can we let ActiveRecord handle this for us?

```suggestion
    where.not(identifier: REPLACEMENT_SCHEDULE_IDENTIFIERS)
```